bool toFilter
float64 deltaT
float64 cutoffFrequency
---
